source/features/close-out-of-view-modals.tsx

```diff
@@ -11,10 +11,10 @@ const observer = new IntersectionObserver(([{intersectionRatio, target}]) => {
 
 function init(): void {
 	// The `open` attribute is added after this handler is run, so the selector is inverted
-	delegate('.details-overlay:not([open]) > summary', 'click', ({delegateTarget: summary}) => {
+	delegate('.details-overlay:not([open]) > summary[aria-haspopup="menu"]', 'click', ({delegateTarget: summary}) => {
```

Comment:
I'd avoid this extra check, there's only ever one `summary`

Reply:
Nope, Let me get you a few links

Reply:
Indeed, hopefully GitHub settled on `[aria-haspopup="menu"]' + details-menu` for all. Maybe we should also make sure all of our dropdown match this as well.

Reply:
In this PR?

Reply:
@fregante Done